introduction.md

```diff
@@ -1,3 +1,6 @@
 hi, my name is jay
 
-my favorite codeschool course has been the Electives
\ No newline at end of file
+my favorite codeschool course has been the Electives
+
+* git
+* python
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?